app/request_utils.py

```diff
@@ -0,0 +1,5 @@
+def generate_request_id() -> str:
+    from random import randbytes
+    from base64 import b64encode
```

Comment:
Why not top level import?

Reply:
Total overlook, probably mistake rearranging lines. Fixed it!